test/unit_test/automl_test.cc

```diff
@@ -605,6 +610,172 @@ BOOST_AUTO_TEST_CASE(one_diff_impl_unittest_w_iterations)
       test_hooks, num_iterations, seed);
 }
 
+BOOST_AUTO_TEST_CASE(qbase_unittest_w_iterations)
```

Comment:
New tests should be added to `vowpalwabbit/core/tests`

Reply:
I'll migrate all the file, since this is almost an exact copy of another test on this same file.